internal/server/data/migrations.go

```diff
@@ -50,8 +60,8 @@ func migrations() []*migrator.Migration {
 		addAuthURLAndScopeToProviders(),
 		setDestinationLastSeenAt(),
 		deleteDuplicateGrants(),
-		addFieldsFor_0_14(),
 		dropDeletedProviderUsers(),
+		addFieldsFor_0_14(),
```

Comment:
Normally re-ordering these is not good, but this was added in a not-yet-merged PR, so no risk here.